### DISCUSSION TITLE
Unix socket support

### DISCUSSION THREAD

**Discussion by mrene** [routerify/routerify#61] [Show and tell]
I was looking at using hyperlocal with routerify, which almost worked but since the connection type is different, the `RouterService` impl did not match. I initially looked into adding it directly to routerify, but certain things made it difficult to have a clean in-tree implementation (the biggest one being an IP `SockerAddr` not being optional).

Since hyper's support is done through [the hyperlocal crate](github.com/softprops/hyperlocal) it made sense to have a separate crate for routerify's support as well, [so here it is](https://github.com/mrene/routerify-unixsocket)

Is there a list somewhere where people can add related crates?

**Comment by gsserge** [comment]
I don't think there's a list. How about putting it under this (routerify) organization?

**Reply by mrene** [reply]
That makes sense, I'll add you to the repo so you can transfer it.

**Reply by mrene** [reply]
Well TIL that with personal repos, you can't have more than one admin. You can either temporarily grant me access to create repos in the `routerify` org, or I can transfer it to your user and you can then transfer it to the org.

**Reply by gsserge** [reply]
I've accepted the invite, but I can't access the repository Settings to make a transfer.

I've never done a repo transfer before, but looking at the github documentation, it's my understanding that you can initiate the transfer as the repo owner, specify `routerify` org as a recipient and I will try to accept it (I'm not an owner of the routerify org myself).

**Reply by mrene** [reply]
I'll transfer it to you then.

**Reply by mrene** [reply]
You can't transfer to an org you don't have _create_ access to, only to another user.

**Reply by gsserge** [reply]
Got it. The repo is now under the routerify org. Unfortunately I do not have enough permissions to add you as a collaborator. 

@rousan can you please add @mrene as a collaborator?

**Reply by gsserge** [reply]
Oh nice, after the transfers you still have access to the repo, so you are not blocked.

**Reply by mrene** [reply]
Yeah I'm just missing admin access but I can still push code. I've updated the references and re-published the crate with the updated repository url.

**Reply by rousan** [reply]
Adding @mrene as collaborator to routerify org.

**Reply by rousan** [reply]
@mrene, I have sent you a Github invitation.
